doc/source/structures/orbits/orbit.rst

```diff
@@ -17,6 +17,31 @@ Whenever you get the :struct:`Orbit` of a :struct:`Vessel`, be aware that its ju
 
     Some of the parameters listed here come directly from KSP's API and there is a bit of inconsistency with whether it uses radians or degrees for angles. As much as possible we have tried to present everything in kOS as degrees for consistency, but some of these may have slipped through. If you see any of these being reported in radians, please make a bug report.
 
+
+Creation
+--------
+
+.. function:: ORBIT(x, v, body, t)
+
+    :parameter x: (vector) position at time t in the :ref:`ship-center-raw-rotation <ship-raw>` frame
+    :parameter v: (vetor) velocity at time t
+    :parameter body: (CelstialBody) central body of orbit
+    :parameter t: (scalar) universal time
+    :return: :struct:`Orbit`
+
+    This creates a new user defined orbit (for predictive/calculation purposes) around a body given the position and velocity
+    at a given time.  The vectors are in the ref:`ship-center-raw-rotation <ship-raw>` frame.  This makes it easy to
+    recover (or perturb) an orbit based on a given position and velocity:
+
+        SET t TO TIME:SECONDS + 100.
+        SET o TO ORBIT( obt:velocityat(t):orbit, obt:positionat(t), body, t ).
+
+    Here, a new :struct:`Orbit` called ``o`` is created that should be very nearly equal to the current orbit.
+
+    To create an orbit based on the position (r) and velocity (v) from the center of the body a translation is required:
+
+        SET o TO ORBIT( r + body:position, v, body, TIME:SECONDS ).
+
```

Comment:
I think if we are going to provide a way to built an Orbit object directly from parameters without a vessel, we should probably also have a second version of this constructor that does it by Keplerian parameters rather than by position/velocity.   I'm thinking of the contracts in which you are told to get a satellite into a specific orbit, and you are told the parameters of that orbit.  It might be useful for people to be able to construct that orbit object from those parameters.